airflow/docker-compose.yaml

```diff
@@ -101,6 +101,7 @@ x-airflow-common:
     POD_LOCATION: "us-west2-a"
     AIRFLOW_ENV: "development"
     CALITP_USER: "pipeline"
+    SENTRY_DSN: "https://e498431022154366b0ff8b71cf2d93e0@sentry.calitp.org/2"
```

Comment:
is this DSN dedicated to local envs? Highly recommend using separate DSNs to segment traffic between envs as most of Sentry's early-stage filtering and flow control is at the DSN level. I've hosed DSNs before by letting them float around to local dev envs

Reply:
Does that imply separate projects? I was planning to segment by environment rather than project if possible. I can certainly split off a dev project.

Reply:
I'm going to deploy this as-is but make myself an issue to investigate separate envs/DSNs. Mainly I wonder how it affects team/member issue assignment and notifications.